### PR TITLE
Doctor Aphra

### PR DESCRIPTION
**Pull request by bwohldmann**
image

### CODE REVIEW

File: test/server/cards/07_LAW/units/DoctorAphraDiggingForAnswers.spec.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                }
+            });
+
+            const { context } = contextRef;
+
+            context.player1.clickCard(context.doctorAphraDiggingForAnswers);
+            context.player1.clickCard(context.p2Base);
+
+            expect(context.player2).toBeActivePlayer();
+        });
```
Comment (AMMayberry1) [inline test/server/cards/07_LAW/units/DoctorAphraDiggingForAnswers.spec.ts]: Let's add a test where the deck is empty just to make sure nothing breaks
